src/nvim/window.c

```diff
@@ -700,9 +700,10 @@ int win_fdccol_count(win_T *wp)
   const char *fdc = (const char *)wp->w_p_fdc;
 
   // auto:<NUM>
-  if (strncmp(fdc, "auto:", 5) == 0) {
+  if (strncmp(fdc, "auto", 4) == 0) {
+    const int fdccol = fdc[4] == ':' ? fdc[5] - '0' : 1;
```

Comment:
Is this the reason for supporting up to 9 columns only?

Reply:
yeah, I couldn't imagine anyone using 12 foldlevels. We haven't received any complaint so far but if you want to support 12 columns go ahead